src/main/resources/application-local.yaml

```diff
@@ -6,27 +6,21 @@ spring:
   boot:
     admin:
       ui:
-        hide-instance-url: true
+        hide-instance-url: false  # Show URLs in local development
```

Comment:
when you are using docker these urls are useless because they point to docker internal network, when you click then always point to a broken link